### PR TITLE
Test check quota cache hit

### PR DESCRIPTION
**Pull request by JimmyCYJ**
Fixes #[1181](https://github.com/istio/proxy/issues/1181)

### CODE REVIEW

File: mixer/test/client/auth/auth_test.go [hunk 1]
```diff
@@ -112,6 +112,8 @@ const reportAttributes = `
   "target.uid": "POD222",
   "target.namespace": "XYZ222",
   "connection.mtls": false,
+  "check.cache_hit": false,
+	"quota.cache_hit": false,
```
Comment (qiwzhang) [inline mixer/test/client/auth/auth_test.go]: please fix indentation

File: mixer/test/client/check_cache_hit/check_cache_hit_test.go [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+    ":authority": "*",
+    "x-forwarded-proto": "http",
+    "x-istio-attributes": "-",
+    "x-request-id": "*"
+  }
+}
+`
+
+// Report attributes from a good GET request
+var reportAttributesOkGet = [...]string{`
```
Comment (qiwzhang) [inline mixer/test/client/check_cache_hit/check_cache_hit_test.go]: please add comment on the different betwen [0] and [1]

File: mixer/test/client/check_cache_hit/check_cache_hit_test.go [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+  "source.namespace": "XYZ11",
+  "source.ip": "[127 0 0 1]",
+  "source.port": "*",
+  "target.name": "target-name",
+  "target.user": "target-user",
+  "target.uid": "POD222",
+  "target.namespace": "XYZ222",
+  "connection.mtls": false,
+	"check.cache_hit": true,
+	"quota.cache_hit": true,
```
Comment (qiwzhang) [inline mixer/test/client/check_cache_hit/check_cache_hit_test.go]: please fix all indentations.

File: mixer/test/client/check_cache_hit/check_cache_hit_test.go (hunk 2 above)
Comment (JimmyCYJ) [reply mixer/test/client/check_cache_hit/check_cache_hit_test.go]: Done. Thanks.

File: mixer/test/client/check_cache_hit/check_cache_hit_test.go (hunk 3 above)
Comment (JimmyCYJ) [reply mixer/test/client/check_cache_hit/check_cache_hit_test.go]: Done. Thanks.

File: mixer/test/client/auth/auth_test.go (hunk 1 above)
Comment (JimmyCYJ) [reply mixer/test/client/auth/auth_test.go]: Done. Thanks.

**Comment by JimmyCYJ** [comment]
Please take a look. Thanks.

**Pull request merged**